src/Illuminate/Http/Request.php

```diff
@@ -255,6 +255,24 @@ public function has($key)
 	}
 
 	/**
+	 * Determine if the request contains a non-empty value for an input item matching a given pattern.
+	 *
+	 * @param  string  $pattern
+	 * @return bool
+	 */
+	public function hasRegex($pattern)
+	{
+		$input = $this->all();
+
+		foreach ($input as $key => $value)
```

Comment:
I wonder if `$input` variable is needed here ? Maybe you can write this as `foreach($this->all() as $key => $value)`. It's my first comment in this repo so go easy on me :see_no_evil:.